<?php
/**
 * Created by PhpStorm.
 * User: wangzhongjiang
 * Date: 15/6/28
 * Time: 下午1:23
 */

namespace EZPHP;


class dev extends  base{
    public static function start(){
        //todo  时间 内存


        //难道把代码放到 try 里面去？
//        try {
//            $o = new TestException(TestException::THROW_CUSTOM);
//        } catch (MyException $e) {      // 捕获异常
//            echo "Caught my exception\n", $e;
//            $e->customFunction();
//        } catch (Exception $e) {        // 被忽略
//            echo "Caught Default Exception\n", $e;
//        }

    }

    public static function end(){
        //todo  时间内存




    }

}

?>
